models/quickbooks__ap_ar_enhanced.sql

```diff
@@ -143,7 +145,18 @@ final as (
             else 0
                 end as days_overdue,
         invoice_join.initial_payment_date,
-        invoice_join.recent_payment_date
+        invoice_join.recent_payment_date,
+        invoice_join.total_current_payment as total_current_payment
+
+        {% else %}
+        ,
+        null as is_overdue,
```

Comment:
These changes address the downstream dependency issue when `using_payments` is disabled. Currently I have it so these impacted fields are null if `using_payment` is disabled.